Tools/CreateBulkConnections_ConfCons2_6.ps1

```diff
@@ -1,114 +1,237 @@
 ﻿#####################################
-# Author: David Sparer
-# Summary: 
+# Authors: David Sparer & Jack Denton
+# Summary:
 #   This is intended to be a template for creating connections in bulk. This uses the serializers directly from the mRemoteNG binaries.
 #   You will still need to create the connection info objects, but the library will handle serialization. It is expected that you
 #   are familiar with PowerShell. If this is not the case, reach out to the mRemoteNG community for help.
 # Usage:
-#   Replace or modify the examples that are shown toward the end of the script to create your own connection info objects. 
+#   Replace or modify the examples that are shown toward the end of the script to create your own connection info objects.
 #####################################
 
-$EncryptionKey = (Get-Credential -Message "Enter the encryption key you would like to use. This must match the encryption key used by the rest of the confCons file." -UserName "DontNeedUsername").Password
-$PathToMrngFolder = ""
-
-if ($PathToMrngFolder -eq "") {
-    Write-Error -Message 'You must set the $PathToMrngFolder variable in this script to the folder which contains mRemoteNG.exe'
+foreach ($Path in 'HKLM:\SOFTWARE\WOW6432Node\mRemoteNG', 'HKLM:\SOFTWARE\mRemoteNG') {
```

Comment:
This will work for installed versions of mRemoteNG, but portable versions will not have a registry entry. If no version was installed on the machine, the later assembly load will fail. Could a prompt be added if no reg entry is found?

Reply:
Ah didn't realise you had a portable version! Yeah, I'll add a prompt for if no installed version is found 🙂 Do you want me to squash the commits to tidy them up or you OK to have minor fixes in the commit history?

Reply:
We are ok with minor fixes like that, no need to squash.